### ISSUE TITLE
Applied Energistics/General Autocrafting issue

### ISSUE THREAD

**Issue by Hundifan33** [TheLimePixel/Modern-Skyblock-3#798]
Can't autocraft the certus quartz seeds in the ME-System or any other autocrafting machine, I have already tried to remove that one line in stages.zs , but it didn't help (I also didn't get any kind of error about that in the log)

1. Try inputting the pattern in the Pattern Terminal and it will show no result and if you encode it, it will show up as an invalid Pattern

**Pack Information**
 - Pack version: 3.5.0
 - Pack Mode: Staged
 - SP or MP: MP
 - Added Mods: Not enough Wands

Additional context:
Tried to remove the line mentioned in #636 (it is in a different line now) but that didn't help. There seems to be no similar error in the log. Also tried the enderio and the factorytech assembler. Other Items (that seem to not be locked) work fine.

**Comment by pkpn** [comment]
I have the same issue on my server. However it seems to be the problem with all the staged crafting recipes like #636 talk about. The lines in stages.zs isn't the places like in #636 either and has no asterix (*) but numbers instead (10, 50, 70).
I managed to get the Enderio staged crafting recipes to work in the Enderio crafters though. (Basic Capacitor, Simple Machine Chassis and Infinity Bimetal Gear) - the other staged recipes makes the Crafter pull the items but the output just vanishes.

**Comment by Hundifan33** [comment]
I already tried removing those lines too, but got no reaction to that.

**Comment by pkpn** [comment]
I have tried that too to no avail. I was merely talking about what I had accomplished for you to maybe try... About the Enderio staged crafting recipes working with the Enderio crafter if you set the Pattern to Processing instead of crafting. This doesn't work with other recipes than the 3 I listed previously though. At least to my knowledge. :)

**Comment by pkpn** [comment]
Does anyone know what lines to out comment in stages.zs now a days to make this work? Anyone from the original #636 thread perhaps?

Link to pastebin of my stages.zs file: https://pastebin.com/mp8nXeka

**Comment by Hundifan33** [comment]
I dont think that removing lines in stages.zs will help, because like I said, I already deleted the lines that where known for causing this problem in an old version of the pack, but it didnt make the autocrafting work. I also didnt find any error in my new and old logs that indicate any similarity to the error that occured in #636 .

**Comment by CDrac0** [comment]
i added refined storage to the mod and wrote this line at the end of stages.zs 

mods.recipestages.Recipes.setPackageStage("com.raoulvdberge.refinedstorage",  ["br", "bt", "mf", "bf", "ba", "bs", "stm", "abf", "amc", "aa", "tpos", "litf", "hm", "e", "b", "atm", "up", "rt", "oa", "ub", "tm", "c", "ir", "am", "tsab", "ttr", "te", "lm", "ds", "et", "ac", "ar", "rft", "ad", "mc", "tpod", "ee", "o", "eu", "atb", "em","tw"]);

if you go with this method backup your better questing files before
and repaste them after you load your save once

**Comment by muvo24** [comment]
Is there a fix for this? The solution in #636 didn't work for me.

**Comment by EricBuist** [comment]
That's quite concerning. When I started to get issues with auto-crafting,, I tried in a test world with AE2 (which I didn't reach yet) and was able to auto-craft a Thermal Expansion machine frame, which is a staged recipe. Now Certus quartz seeds fails to auto craft, I didn't try that, but that shows some staged recipes work, others don't. In stages.zs, there are lines allowing some mods to bypass the GameStages. These are the setPackageStages lines. Looking at those lines, appeng (probably Applied Energistics) can craft for pretty much all stages, including ds where Certus Quartz seeds belongs. If Certus Quartz seeds belongs to another stage by any chance, check if it is in the list for appeng.
The EnderIO crafter is probably in the crazypants package. That one has access to far less stages, excluding ds, so no chance ever to  auto-craft Certus quartz seeds with EnderIO, unless you modify stages.zs.
The other containers are also pretty weak in terms of stages so early game auto-crafting options will certainly be limited. The best hope is AE2.
It's possible that companion mods for AE2, especially wireless crafting terminal, have different package names and would need a line in stages.zs, but I don't know how to figure out the package name.

If AE2 auto-crafting still fails, either the package name appeng doesn't correspond anymore to AE2, either the stages.zs is not read completely, either GameStages has bugs causing the configuration not to work. In any case, this kind of completely breaks the pack, because without auto-crafting, some of the quests will be extremely time consuming, e.g., the top tier storage cell from Extra Cells, and the higher tier TechReborn machines.

The only workaround I could think of is to remove lines from stages.zs, but you'll end up removing pretty much everything and would then be allowed to craft everything, bypassing the progression like in kitchensink mode. Moreover, it's possible that claiming trophies for any unlocked stage will trigger an error because the stages will not exist anymore.
